src/shared/containers/admin/advanced.jsx

```diff
@@ -0,0 +1,286 @@
+            <div style={infoStyleD}>
+              Backend configuration
+              <Button
+                raised
+                disabled={saveBackendDisabled}
+                style={{ float: "right" }}
+                onClick={(e) => {
+                  e.preventDefault();
+                  this.onSaveBackend();
+                }}
+              >
+                SAVE
+              </Button>
+            </div>
+            <form style={infoStyleD}>
+              <div style={{ width: "520px" }}>
+                <TextField
+                  onChange={() => {}}
+                  label="Public Api url"
+                  style={{ width: FORM_WIDTH }}
+                  defaultValue={backend.publicUrl}
+                />
+                <Icon
+                  /* colored={hasTunnelParams} */
+                  style={{ float: "right", marginTop: "8px" }}
+                  name="link"
+                  onClick={(e) => {
+                    e.preventDefault();
+                    this.displayTunnelDialog();
+                  }}
+                />
+              </div>
+              <div>
+                <TextField
+                  onChange={() => {}}
+                  label="Api url"
+                  disabled
+                  style={{ width: FORM_WIDTH }}
+                  defaultValue={backend.apiUrl}
+                />
+              </div>
+              <div>
+                <TextField
+                  onChange={() => {}}
+                  label="Auth url"
+                  disabled
+                  style={{ width: FORM_WIDTH }}
+                  defaultValue={backend.authUrl}
+                />
+              </div>
+              <div>
+                <TextField
+                  onChange={() => {}}
+                  label="AppId"
+                  disabled
+                  style={{ width: FORM_WIDTH }}
+                  defaultValue={backend.clientId}
+                />
+              </div>
+              <div>
+                <TextField
+                  onChange={() => {}}
+                  label="Secret"
+                  disabled
+                  style={{ width: FORM_WIDTH }}
+                  defaultValue={backend.clientSecret}
+                />
+              </div>
+            </form>
+            <div />
+          </Cell>
+        </Inner>
+        <Inner>
+          <Cell className="mdl-color--white" span={12}>
+            <div style={infoStyleD}>
+              Email server configuration
+              <Button
+                raised
+                disabled={saveEmailDisabled}
+                style={{ float: "right" }}
+                onClick={(e) => {
+                  e.preventDefault();
+                  this.onSaveEmailParams();
+                }}
+              >
+                SAVE
+              </Button>
+            </div>
+            <form style={infoStyleD} autoComplete="nope">
+              <div>
+                <TextField
```

Comment:
We could have some basic validation ?
![screen shot 2018-05-24 at 16 13 33](https://user-images.githubusercontent.com/395718/40491119-87ac9ed0-5f6d-11e8-91c0-5c35e65e088a.png)

Reply:
I add them but they are in zoapp-backend and you don't use this version :-) https://github.com/Zoapp/backend/pull/35

Reply:
done